tools.js

```diff
@@ -311,6 +311,7 @@ module.exports = (function () {
 				else move.gen = 0;
 			}
 			if (!move.priority) move.priority = 0;
+			if (!move.ignoreImmunity) move.ignoreImmunity = (move.category === 'Status' ? true : {});
```

Comment:
I'm not exactly a huge fan of instantiating a new Object every time we want to represent "this move doesn't ignore immunities". Why not `false`?